src/game_objects/page_manager.py

```diff
@@ -1,11 +1,12 @@
+from lib import event_manager
 from lib.game_object import GameObject
 from game_objects.views.page_manager_view import PageManagerView
 from game_objects.page import Page
 from game_objects.page_slot import PageSlot
 
-
 class PageManager(GameObject):
     _TOTAL_ROWS = 11
+    _NUM_RAM_COLS = 16
```

Comment:
added class constant (and methods below) to be used from scene.Game to pass to the automated script